packages/babel-preset-mc-app/index.js

```diff
@@ -10,6 +10,7 @@ module.exports = function getBabePresetConfigForMcApp() {
   const isEnvDevelopment = env === 'development';
   const isEnvProduction = env === 'production';
   const isEnvTest = env === 'test';
+  const isRollup = process.env.BUILD_ROLLUP === true;
```

Comment:
Build tool sniffing feels a bit odd but I can not think of anything else other than solving the issue somewhere entirely different.